RobotCode/src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.ArmConstants;
+import frc.robot.subsystems.ArmSubsytem;
+
+public class ArmCommand extends Command
+{
+	public ArmSubsytem arm;
+	public double targetAngle;
```

Comment:
targetAngle is never set, it needs to be passed into the constructor and set there